src/main/java/com/google/sps/servlets/NotificationServlet.java

```diff
@@ -0,0 +1,195 @@
+// Copyright 2019 Google LLC
```

Comment:
All of the code in here seems unrelated to mentor approval tables, which is the title of this PR. Are these changes intended for this PR?

Reply:
I had to pull Andrés's branch, but they're now in master, so this should not be reflected as a change.